prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -459,6 +459,9 @@ module dftbp_initprogram
     !> Do we need to show Mulliken charges?
     logical :: tPrintMulliken
 
+    !> Logical to determine whether to calculate net charge per atom (qNetAtom)
+    logical :: tAllocate
```

Comment:
OK, I will use 'is' than 't'. In 3512 line, the logical 'tNetAtomCharges' already exists to decide printing of qNetAtom. How about 'isDispMbd'? Actually, qNetAtom is used when the user want to use MBD (isDispMbd) or to print net atom charge (tNetAtomCharges) to detailed.out.

Reply:
isQNetAllocated might still be a good use, as in principle this can be true even if MBD is not used.
It looks like the intel 19 compiler, when compiled with debug enabled is failing regression tests for dftb+_reks/CNH4+_dftd4* can you test that combination, or should I look at this?

Reply:
I see, I will change to 'isQNetAllocated'.
For regression tests, I will test them.

Reply:
Currently, with intel 2021.1 compilers, the compilation is failed with debug enabled showing following sentence.

"catastrophic error: **Internal compiler error: internal abort** Please report this error along with the circumstances in which it occurred in a Software Problem Report.  Note: File and line given may not be explicit cause of this error."

This error occurs in './build-debug/_deps/mbd-src/src/mbd_methods.F90'.

Reply:
Now, the regression test will be passed, please test once again with fixed code. And please see commit 19d79ab.
The reason why the error occurs are explained in the commit message. With my intel 21 compiler, all tests pass.